### DISCUSSION TITLE
blog/test

### DISCUSSION THREAD

**Discussion by giscus** [steviegt6/tomat.dev#14] [Blog Discussions]
# blog/test

Hello, world!

https://tomat.dev/blog/test

**Comment by NotLe0n** [comment]
$$legalize\ nuclear\ \color{red}bombs$$

**Comment by absoluteAquarian** [comment]
dQw4w9WgXcQ

**Reply by steviegt6** [reply]
Can't believe HowToBasic did a face reveal.

**Comment by steviegt6** [comment]
$x^2+bx+c=0 \implies x=\frac{-b \pm \sqrt{b^2-4c}}{2}$

**Reply by steviegt6** [reply]
$ax^2+bx+c=0 \implies x=\frac{-b \pm \sqrt{b^2-4ac}}{2a}$

**Reply by NotLe0n** [reply]
```math
\large{\textsf{Objectively better formular:}}
```
```math
x^2 + px + q = 0 \Rightarrow x = -\frac{p}{2}\pm\sqrt{\left(\frac{p}{2}\right)^2-q}
```

**Reply by steviegt6** [reply]
legalize nuclear bombs

**Comment by ghost** [comment]
bro... cool site... and blog... ok sayonara

**Reply by steviegt6** [reply]
ok sayonara

**Comment by ScalarVector1** [comment]
Not half bad. Now can you reglorp the site to hook up to the dingleframe and combobulate fingflops in parallel?

**Reply by steviegt6** [reply]
Right after I

**Comment by sonOfArent** [comment]
Write a comment

**Reply by steviegt6** [reply]
Write a reply

**Reply by ScalarVector1** [reply]
![parametric_curve_(4, 5, 6, 7)](https://github.com/steviegt6/tomat.dev/assets/5168952/29980e49-19ed-4241-b6d1-7fe19ed88e5f)
